exonum/src/blockchain/tests.rs

```diff
@@ -61,16 +62,16 @@ fn test_u64() {
 
 #[test]
 fn test_system_time() {
-    use std::time::{SystemTime, UNIX_EPOCH};
+    use std::time::UNIX_EPOCH;
     encoding_struct! {
         struct Test {
-            some_test: SystemTime,
+            some_test: DateTime<Utc>,
```

Comment:
What do you think about to create a type aliasing like: `type DateTimeUtc = DateTime<Utc>` and use it everywhere in the code?

Reply:
Hmmm, it doesn't seem worth it to me

Reply:
I'm not sure. Are there other opinions?

Reply:
As for me, type alias makes sense if it is more concise (for example, `DateTime` instead of `DateTime<Utc>`) or it adds more contextual meaning (for example `TransactionTime`).

I'm happy with DateTime<Utc>  by the way.

Reply:
I don't like `DateTime`, because if user use `chrono::DateTime` in his code, he will face errors like "DateTime is already imported into the scope". What he gonna do? Import it like `use chrono::DateTime as RealDateTimeNotProvidedWithExonumOne` :smile: ?
I'm also against `TransactionTime`, as it's completely wrong in terms of semantics.

Reply:
`TransactionTime` was just an example of a "meaningful" name in some context. :upside_down_face: